### ISSUE TITLE
Aria-label needs to be added on ibm-select component

### ISSUE THREAD

**Issue by Shoeb009** [IBM/carbon-components-angular#1650]
## Detailed description

> Describe in detail the issue you're having.
* in DAP test getting a violation for ibm Select component as Form control element **Select** has no associated label
> Is this a feature request (new component, new icon), a bug, or a general issue?
* general issue

> Is this issue related to a specific component?
* yes, ibmSelect

> What did you expect to happen? What happened instead? What would you like to see changed?
* there should not be violation w.r.t to ibmSelect component, got a violation with missing label, i would like to see no violation with carbon component

> What browser are you working in?
* chrome

> What version of the Carbon Design System are you using?
* "carbon-components-angular": "^4.13.2"
* "carbon-components": "^10.18.0"

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
* I work with  zStore application in WSE team,  application should go to UAT in coming week

## Additional information

* Screenshots or code
<img width="931" alt="dap-customer err" src="https://user-images.githubusercontent.com/40066507/101444231-199a4780-3945-11eb-9522-0891598fc50a.png">

**Comment by Shoeb009** [comment]
Hi @zvonimirfras,

**Comment by zvonimirfras** [comment]
Hi @Shoeb009, no timeline at the moment.
If you know what the fix is and can contribute it, we would appreciate it and that would greatly speed up the process 🙂

**Comment by Shoeb009** [comment]
Hi @zvonimirfras , am not 100% sure about the fix but i would like to give a try to resolve this issue and contribute it, could you please share the process to do so ?
